test/Yarn/V2/testdata/yarn.lock-e2e

```diff
@@ -0,0 +1,92 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
+
+__metadata:
+  version: 4
+  cacheKey: 8
+
+"acorn-jsx@npm:^3.0.0":
```

Comment:
[optional] [nit] Can you add git sourced, url sourced, archive sourced, and directory sourced dependency with yarn-v2 to include them on e/e test?

Reply:
I think those tests are in the resolver tests.  This is meant to test the graphing logic in full.